ops/setup.py

```diff
@@ -23,11 +23,11 @@
     name="ops.interface_kube_control",
     packages=find_namespace_packages(include=["ops.*"]),
     url="https://github.com/juju-solutions/interface-kube-control/blob/HEAD/ops",
-    version="0.2.0",
+    version="0.3.0",
     zip_safe=True,
     install_requires=[
         "backports.cached-property",
-        "pydantic<2",
+        "pydantic",
         "ops",
```

Comment:
Is there anyway here we can confirm this same code works on `pydantic<2` and `pydantic >=2`

Maybe two different unit tests runs?

Reply:
This code won't run in `pydantic==1.*`. We have introduced new APIs like `model_dump` and `core_schema` which are only available in the latest version. Should we create a new branch which only tracks pydantic 2?

Reply:
I created a tag `ops-0.2.0` to reference if we need to patch the pydantic==1.* version of this repo